lib/Recorder.cpp

```diff
@@ -535,6 +535,17 @@ void Recorder::recordGenericSet(
     recordLine("s|" + key + "|" + Globals::joinFieldValues(arguments));
 }
 
+void Recorder::recordGenericCounterPolling(
+        _In_ const std::string& key,
+        _In_ const std::vector<swss::FieldValueTuple>& arguments)
+{
+    SWSS_LOG_ENTER();
+
+    // lower case 'p' stands for counter Polling
+
+    recordLine("p|" + key + "|" + Globals::joinFieldValues(arguments));
```

Comment:
can you also add support for this is saiplayer ? and also add a sample test to replay sample file that contain this kind of logic example you can takie from here: https://github.com/sonic-net/sonic-buildimage/issues/20576

Reply:
and example test to replay you can take from here: https://github.com/sonic-net/sonic-sairedis/blob/master/tests/BCM56850.pl#L756

Reply:
Unlike SAI API call, this is to spawn a new thread. I'm not sure whether it's doable in saiplayer infra.
let me check it.

Reply:
you can add support 'p' and just ignore it in saiplayer for now, just to make sure it will replay asic  configuration correctly

Reply:
will do it in this way